_config.yml

```
@@ -1,5 +1,4 @@
-theme: jekyll-theme-minimal
+theme: jekyll-theme-cayman
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)